src/spath.c

```diff
@@ -1423,25 +1409,25 @@ Pretty print for TotalView debug window
 /* This enables a nicer display when diving on a path variable
  * under the TotalView debugger.  It requires TV 8.8 or later. */
 
-#include "tv_data_display.h"
+//#include "tv_data_display.h"
 
-static int TV_ttf_display_type(const spath* path)
-{
-  if (path == NULL) {
+//static int TV_ttf_display_type(const spath* path)
```

Comment:
@kosinovsky: oops, yeah, sorry. I was referring to these lines that were commented out as well (between lines 1402-1433)